### ISSUE TITLE
Tauri v2 compatiblity

### ISSUE THREAD

**Issue by dan-myles** [oscartbeaumont/tauri-specta#83]
Hey, just opening something up asking about tauri v2 compatability, seems like it just doesn't work as of now.

Any word?

**Comment by oscartbeaumont** [comment]
I'm keeping track of the plan [over here](https://github.com/specta-rs/tauri2), but we are currently waiting on the Tauri team to merge [a PR](https://github.com/tauri-apps/tauri/pull/9392) and due to my personal lack of time to work on this.

My GitHub Sponsors is open if you really need this.

I am closing as a duplicate of #62.
